utils/constants.jsx

```diff
@@ -1072,10 +1072,10 @@ export const Constants = {
     HighlightedLanguages: {
         actionscript: {name: 'ActionScript', extensions: ['as'], aliases: ['as', 'as3']},
         applescript: {name: 'AppleScript', extensions: ['applescript', 'osascript', 'scpt']},
-        bash: {name: 'Bash', extensions: ['bash', 'sh', 'zsh']},
+        bash: {name: 'Bash', extensions: ['bash', 'sh', 'zsh'], aliases: ['bash', 'sh']},
```

Comment:
Is bash necessary as an alias if it's already a listed extension? Looks like it already works on master

Reply:
I don't know the difference between `extensions` and `aliases` i made the same as other languages (`actionscript` for example).

Reply:
`extensions` is used to trigger previews of attached source files while `aliases` is used to provide alternate names of the language when using a code block. There's a couple things we might want to change here:
1. I don't know if people use `.bash` or `.zsh` file extensions for script files, so we might be able to remove those from `extensions`
2. You can remove `bash` from the `aliases` since we also use the key for each entry as a possible language name.